pkgs/applications/audio/mopidy/podcast.nix

```
@@ -0,0 +1,26 @@
+{ lib, python3Packages, mopidy }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mopidy-podcast";
+  version = "3.0.0";
+
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "Mopidy-Podcast";
+    sha256 = "1z2b523yvdpcf8p7m7kczrvaw045lmxzhq4qj00dflxa2yw61qxr";
+  };
+
+  propagatedBuildInputs = [
+    mopidy
+    python3Packages.cachetools
+    python3Packages.uritools
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://www.mopidy.com/";
```

```suggestion
    homepage = "https://github.com/tkem/mopidy-podcast";
```

This is the actual homepage. If you visit their pypi page it links there.